tests/src/integration/tests/test_set_keyspace.cpp

```diff
@@ -175,7 +175,7 @@ CASSANDRA_INTEGRATION_TEST_F(SetKeyspaceTests, QueryNotSupported) {
  */
 CASSANDRA_INTEGRATION_TEST_F(SetKeyspaceTests, QueryWithNoSessionKeyspace) {
   CHECK_FAILURE;
-  CHECK_VERSION(4.0.0);
+  CHECK_PROTOCOL_VERSION(CASS_PROTOCOL_VERSION_V5);
```

Comment:
The changes above mean that we're now _not_ trying to connect as a beta version of v5.  This means that for C* 4.0 we'll now connect with v4 (the max version actually supported by this driver).  This change is actually a problem for some tests (like this one) which rely on v5 features [1] but can't be executed against an actual server since we only now have v5 support.

To be more explicit about these requirements we constrain these tests by protocol version used for a given connection rather than by a Cassandra version.

[1] Set keyspaces for statements was added in protocol v5